build.sh

```diff
@@ -23,9 +28,12 @@ cd wls-module
 docker build -t wls:0.0.1 .
 cd ..
 
-cd analysis-module
-docker build -t analysis .
-cd ..
+# Prepare container proc monitoring venv
+python3 -m venv ./monitoring/container-proc/venv --without-pip
+source ./monitoring/container-proc/venv/bin/activate   
+curl https://bootstrap.pypa.io/get-pip.py | python
+pip3 install -r ./monitoring/container-proc/requirements.txt
+deactivate
```

Comment:
Where you able to run an entire workflow with this?

Since the venv that you are crating is being created inside the kurtosis folder, `kurtosis run` is failing due to the `exceed or reach 4mb, a limit imposed by gRPC.` error. We have to create it outside the kurtosis folder.